packages/cli/test/unit/util/dev/validate.test.ts

```diff
@@ -285,4 +285,78 @@ describe('validateConfig', () => {
 
     expect(error!.link).toEqual('https://vercel.link/functions-and-builds');
   });
+
+  it('should error when crons have missing schedule', () => {
+    const error = validateConfig({
+      // @ts-ignore
+      crons: [{ path: 'api/test.js' }],
+    });
+    expect(error!.message).toEqual(
+      'Invalid vercel.json - `crons[0]` missing required property `schedule`.'
+    );
+    expect(error!.link).toEqual(
+      'https://vercel.com/docs/concepts/projects/project-configuration#crons'
+    );
+  });
+
+  it('should error when crons have missing path', () => {
+    const error = validateConfig({
+      // @ts-ignore
+      crons: [{ schedule: '* * * * *' }],
+    });
+    expect(error!.message).toEqual(
+      'Invalid vercel.json - `crons[0]` missing required property `path`.'
+    );
+    expect(error!.link).toEqual(
+      'https://vercel.com/docs/concepts/projects/project-configuration#crons'
+    );
+  });
+
+  it('should error when path is too long', () => {
+    const error = validateConfig({
+      crons: [{ path: '/' + 'x'.repeat(512), schedule: '* * * * *' }],
+    });
+    expect(error!.message).toEqual(
+      'Invalid vercel.json - `crons[0].path` should NOT be longer than 512 characters.'
+    );
+    expect(error!.link).toEqual(
+      'https://vercel.com/docs/concepts/projects/project-configuration#crons'
+    );
+  });
+
+  it('should error when schedule is too long', () => {
+    const error = validateConfig({
+      crons: [{ path: '/', schedule: '*'.repeat(257) }],
+    });
+    expect(error!.message).toEqual(
+      'Invalid vercel.json - `crons[0].schedule` should NOT be longer than 256 characters.'
+    );
+    expect(error!.link).toEqual(
+      'https://vercel.com/docs/concepts/projects/project-configuration#crons'
+    );
+  });
+
+  it('should error when path is empty', () => {
+    const error = validateConfig({
+      crons: [{ path: '', schedule: '* * * * *' }],
+    });
+    expect(error!.message).toEqual(
+      'Invalid vercel.json - `crons[0].path` should NOT be shorter than 1 characters.'
+    );
+    expect(error!.link).toEqual(
+      'https://vercel.com/docs/concepts/projects/project-configuration#crons'
+    );
+  });
+
+  it('should error when schedule is too short', () => {
+    const error = validateConfig({
+      crons: [{ path: '/', schedule: '* * * * ' }],
+    });
+    expect(error!.message).toEqual(
+      'Invalid vercel.json - `crons[0].schedule` should NOT be shorter than 9 characters.'
+    );
+    expect(error!.link).toEqual(
+      'https://vercel.com/docs/concepts/projects/project-configuration#crons'
```

Comment:
Let's remember to update the documentation after this lands

Reply:
Planned, it had been done already for the previous API in a PR, and we will update that PR